ChangeLog.d/psa-rc4.txt

```diff
@@ -0,0 +1,3 @@
+Bugfix
+   * Fix PSA cipher multipart operations using RC4. Previously, an IV was
```

Comment:
I saw the comment reading typo (`RC4` and `ARC4`) and that both are the same.
Just pointing out that in change log `RC4` is used and in implementation and comments `ARC4` name is used.

Reply:
I amended the last commit to also change that changelog entry.

I didn't bother renaming the file, the file name will disappear once we merge the changelog entries anyway.